pipeline/tasks/protoc_tasks.py

```diff
@@ -359,24 +359,18 @@ def execute(self, language, api_name, output_dir, src_proto_path,
 class JavaGrpcPackmanTask(GrpcPackmanTask):
 
     def execute(self, language, api_name, output_dir, src_proto_path,
-                import_proto_path, packman_flags=None, repo_dir=None):
-        packman_flags = packman_flags or ['--experimental_alt_java']
+                import_proto_path, packman_flags=[], repo_dir=None,
+                dependencies=[]):
+        if len(packman_flags) == 0:
+            packman_flags.append('--experimental_alt_java')
+            for dep in dependencies:
+                packman_flags.append('--dependencies')
```

Comment:
Maybe the packman flag should be singular `--dependency` if it is repeated for each additional dependency